src/services/itemMembership/plugins/MembershipRequest/index.test.ts

```diff
@@ -22,13 +22,19 @@ import { MembershipRequestRepository } from './repository';
 const testUtils = new ItemTestUtils();
 
 function expectMemberRequestToBe(membershipRequest, member?: Member, item?: Item) {
+  expect(membershipRequest.id).toBeUndefined();
```

Comment:
I've removed it from the AJV Schema. This Id is defined in the object that we return, as the PrimaryKey of the SQL Table. There is no use to this Id since we should use the Item Id and the Member Id. I don't think this is a sensitive information but the user should not have any utility to it. This assertion check that AJV is doing his job. (And apparently it does ! 👍🏻 )

Reply:
Okay thanks for your explanation! Maybe just add a very small comment to explain that?